packages/mobile/src/transactions/saga.ts

```diff
@@ -220,6 +282,7 @@ function* addRecipientProfiles({ transactions }: NewTransactionsInFeedAction) {
 function* watchNewFeedTransactions() {
   yield takeEvery(Actions.NEW_TRANSACTIONS_IN_FEED, cleanupStandbyTransactionsLegacy)
   yield takeEvery(Actions.UPDATE_TRANSACTIONS, cleanupStandbyTransactions)
+  yield takeLatest(Actions.UPDATE_TRANSACTIONS, getInviteTransactionsDetails)
```

Comment:
Why not `takeEvery`?

Reply:
hum good question...i was thinking that there shouldn't be multiple UPDATE_TRANSACTIONS actions dispatched in quick succession/in parallel, and if there are then the most recent action would contain the information from previous actions (e.g. the transactions fetched at 10:01am would contain the same transactions fetched at 10:00am) so it's safe to take only the latest action...

but now as i am typing this, i remember that we have paginated transaction feed coming soon so maybe it is safer to do takeEvery after all...